src/common/util.js

```diff
@@ -1,5 +1,5 @@
 function filterRestrictedSymbols(active_symbols) {
-  return active_symbols.filter(function(item) { return !/^(BOOM|CRASH|STP).+/i.test(item.symbol)});
+  return active_symbols;
 }
```

Comment:
Possible to remove the method and its usages across app?

Reply:
yes, since we do not need any symbol filtering we can remove the method, I will do it